charts/onechart/templates/ingress.yaml

```
@@ -38,7 +38,11 @@ spec:
   tls:
     - hosts:
         - {{ .ingress.host | quote }}
+      {{- if .ingress.secretName }}
+      secretName: {{ .ingress.secretName }}
+      {{ else }}
       secretName: {{ printf "tls-%s" $resourceName }}
+      {{- end }}
```

This might also work:

```suggestion
      secretName: {{ .ingress.secretName | default (printf "tls-%s" $resourceName) }}
```